programmers/난이도별/level01.문자열_내_p와_y의_개수/yoojinhee03.java

```diff
@@ -0,0 +1,24 @@
+package programmers;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class yoojinhee03 {
+
+	public static void main(String[] args) {
+		int[] arr= {1,2,3,4,5};
+		System.out.println(solution("pPyyY"));
+		
+	}
+	
+	static boolean solution(String s) {
+
+        s=s.toUpperCase();
+        int cnt=0;
+        String arr[]=s.split("");
+        for(String str:arr) 
+        	cnt= str.equals("P")?cnt+1:str.equals("Y")?cnt-1:cnt+0;  		   
+        if(cnt==0) return true;
+        else return false;

```

Comment:
```suggestion
        return Arrays.stream(s.toUpperCase().split(""))
                     .reduce(0, (cnt, str) => str.equals("P") ? cnt + 1 :
                                              str.equals("Y") ? cnt - 1 : cnt) == 0;
```
스트림을 사용하면 이런식으로 표현할 수 있답니다!

Reply:
스트림을 한 번 공부해봐야겠어요! 리뷰 감사합니다😁